data/src/main/java/io/gnosis/data/repositories/TransactionRepository.kt

```diff
@@ -77,6 +112,24 @@ class TransactionRepository(
                 dataSize = txInfo.dataSize.toLong(),
                 value = txInfo.value.toBigInteger()
             )
+
+            is Creation -> Transaction.Creation(
+                id = id,
+                confirmations = null,
+                status = TransactionStatus.SUCCESS
+            )
+
+            is Unknown -> Transaction.Custom(
+                id = id,
+                address = "0x00".asEthereumAddress()!!,
```

Comment:
why "0x00" and not BigInteger.ZERO? Solidity.Address(BigInteger.ZERO). I think it is more explicit and easier to read

Reply:
Fixed in 9e9bc0cf8afce0f6a2b6a9a4774753181a84ffc1